src/web/app/components/sessions-table/sessions-table.component.html

```diff
@@ -53,18 +53,34 @@
           <tm-ajax-loading *ngIf="sessionsTableRowModel.isLoadingResponseRate" [useBlueSpinner]="true"></tm-ajax-loading>
         </td>
         <td class="actions-cell">
-          <button type="button" class="btn btn-light btn-sm" (click)="editSessionEvent.emit(idx)"
-                  [disabled]="!sessionsTableRowModel.instructorPrivilege.canModifySession">Edit</button>
+          <a *ngIf="sessionsTableRowModel.instructorPrivilege.canModifySession; else editSessionBtn" routerLink="/web/instructor/sessions/edit" [queryParams]="{ courseid: sessionsTableRowModel.feedbackSession.courseId, fsname: sessionsTableRowModel.feedbackSession.feedbackSessionName }">
```

Comment:
Why not just wrap the <a> tag around the `disabled` button?

Reply:
If we wrap the `<a>` tag, the user can still right-click and open link in new tab/middle-click the button since the link is still there which defeats the purpose of the disabled button.